.github/workflows/check_wdk.yml

```
@@ -0,0 +1,62 @@
+    runs-on: Windows-latest
+    steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      if: steps.skip_check.outputs.should_skip != 'true'
+      with:
+        repository: ${{inputs.repository}}
+        submodules: 'recursive'
+        ref: ${{inputs.ref}}
+
+    - name: Check for updates to the Windows Driver Kit
+      id: check_wdk
+      run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)